### PR TITLE
Add All Contributors to the spacefish README.md

### PR DESCRIPTION
**Pull request by matchai**
## Description

To be able to better recognize the contributors to Spacefish, I'd like to add *All Contributors* to the README of our repo.

## Motivation and Context

Improve the recognition of our lovely contributors 💖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Screenshots (if appropriate):
image

## Checklist:

- [x] I have checked that no other PR duplicates mine
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      "name": "Matan Kushner",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/4658208?v=4",
+      "profile": "https://twitter.com/matchai",
+      "contributions": [
+        "bug",
+        "code",
+        "doc",
+        "review",
+        "test",
+        "tool"
```
Comment (tiffafoo) [inline .all-contributorsrc]:
```suggestion
        "tool",
        "design"
```

For myself as well? Even though I just gave suggestions :P

**Review by Snuggle — APPROVED** [review]
Aww, adorable! :two_hearts: 
The `reviewed pull requests` button doesn't seem to work, though? :o

**Review by tiffafoo — APPROVED** [review]
nice gif

**Comment by matchai** [comment]
Yeah, so it seems. I guess there's no easy way within GitHub to filter by reviewer at the moment 🤔

**Pull request merged**